OneDrive-Cloud-Player/Services/Helpers/GraphHelper.cs

```diff
@@ -20,24 +16,57 @@ class GraphHelper
 
         private GraphAuthHelper Auth { get; set; }
 
-        public GraphHelper()
+        private static GraphHelper _instance = null;
+        private static readonly object _instanceLock = new object();
+
```

Comment:
Maybe this one could use a doc. What do you think?

Reply:
Agreed, will add in next commit.